docs/start/configuration/discovery_options.rst

```diff
@@ -42,31 +46,30 @@ You may also specify serial_filter:
        - d073d5001337
        - d073d5001338
 
-In this case, broadcast discovery will be used, but it will only discover those
-two serials and ignore all other devices. This is handy if you want to restrict
-what devices Photons ever interacts with.
+In this case, broadcast discovery is used, but only to discover the specified
+serials. All other devices are ignored.
 
-You can also combine ``serial_filter`` and ``hardcoded_discovery``.
+The ``serial_filter`` and ``hardcoded_discovery`` options can be combined.
 
-You can override what is configured with environment variables. For example,
-regardless of what you have in configuration, if you say something like::
+The ``HARDCODED_DISCOVERY`` environment variable sets or overrides any
+``hardcoded_discovery`` configuration setting. In the following example, Photons
+ will only discover a single device with the serial of ``d073d5111111`` if it
+ has the IP address of ``192.168.0.1``:
 
    $ export HARDCODED_DISCOVERY='{"d073d5111111": "192.168.0.1"}'
    $ lifx lan:get_attr _ color
 
-With this, Photons will only care about ``d073d5111111`` and assume it's at
-``192.168.0.1``
-
-You may also specify ``SERIAL_FILTER``::
+The ``SERIAL_FILTER`` environment variable sets or overrides any ``serial_filter``
+configuration setting::
 
    $ export SERIAL_FILTER=d073d500001,d073d5111111
    $ lifx lan:get_attr _ color
 
-This will do a broadcast discovery, but only care about ``d073d5000001`` and
-``d073d5111111``.
+Photons will do a broadcast discovery, but only for devices with the serial of
```

Comment:
it'll still discover everything else, it'll just pretend the rest of them don't exist